app/src/main/java/com/infomaniak/drive/views/FileInfoActionsView.kt

```diff
@@ -215,7 +215,9 @@ class FileInfoActionsView @JvmOverloads constructor(
                     availableOfflineSwitch.isChecked = false
                     showSnackBarInvalidFileName(currentFile.name)
                 }
-                findNavController().popBackStack()
+                with(findNavController()) {
+                    if (currentBackStackEntry?.destination?.label == "FileInfoActionsBottomSheetDialog") popBackStack()
```

Comment:
Can you put this `FileInfoActionsBottomSheetDialog` in the `strings.xml`, as a non-translatable one?

And use it too in the `main_navigation.xml`.

Reply:
done